test/test_syntax_interface.py

```diff
@@ -664,6 +664,24 @@ def test_graph_txt(self):
         proto.set_model(Model.TestOdeModel(1))
         proto.run()
 
+    def test_non_existent(self):
+        proto_file = 'protocols/not_here.txt'
+        # for some reason this line reports failure (while also reporting the Exception !!
+        # self.assertRaises(FileNotFoundError, fc.Protocol(proto_file))
```

Comment:
If you're using the unittest assertRaises, you need to call it like `self.assertRaises(FileNotFoundError, fc.Protocol, proto_file)`. Or since we're actually running via pytest, we can use `with pytest.raises(FileNotFoundError): ...`

Reply:
okay BUT the rest of the file uses the unittests syntax

Reply:
Yeah, that's because it was written before pytest existed :) I'd like to migrate over gradually, so at least new stuff it written more in the pytest style.